praw/models/reddit/removal_reasons.py

```diff
@@ -73,43 +110,54 @@ def delete(self):
 
         .. code-block:: python
 
-           reddit.subreddit("NAME").removal_reasons["141vv5c16py7d"].mod.delete()
+           reddit.subreddit("NAME").mod.removal_reasons["141vv5c16py7d"].delete()
 
         """
         url = API_PATH["removal_reason"].format(
             subreddit=self.subreddit, id=self.id
         )
-        self.subreddit._reddit.request("DELETE", url)
+        self._reddit.request("DELETE", url)
```

Comment:
Once this is rebased on top of master, you can use the new `.delete()` method.

```suggestion
        self._reddit.delete(url)
```